tests/baselines/reference/thisTypeInFunctionsNegative.errors.txt

```diff
@@ -407,10 +405,6 @@ tests/cases/conformance/types/thisType/thisTypeInFunctionsNegative.ts(178,22): e
                           ~
 !!! error TS1005: ',' expected.
     function decorated(@deco() this: C): number { return this.n; }

```

Comment:
there should still be an error here, right?

Reply:
Aha, yes: the _parser_ should be the one to emit an error. We don't need to wait until the checker...